tutorials/bios-boot-tutorial/README.md

```diff
@@ -17,17 +17,17 @@ The `bios-boot-tutorial.py` script simulates the bios boot sequence.
 
```

Comment:
Not sure the Prerequisites are accurate, maybe just remove them or change it to just be what is needed by the python script itself.

Reply:
ok, let me spin up a docker, and validate the pre-reqs